addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -367,11 +370,15 @@
    <t t-if="queue_type">
         <t t-set="_no_results_title">You've Completely Caught&amp;nbsp;Up!</t>
         <t t-set="result_msg">Go enjoy a cup of coffee.</t>
+        <t t-set="record_name_plural">posts</t>
+        <t t-set="go_back_url" t-valuef="/forum/#{ slug(forum) }/"/>
    </t>
    <t t-else="">
         <t t-set="_no_results_title">Oops!</t>
-        <t t-set="result_msg">Sorry, we could not find any<b>%s</b> results<b>%s</b>%s%s%s.</t>
-        <t t-set="result_msg" t-value="result_msg % (_filters_str.strip(), _my_str.strip(), _search_str.strip(), _search_and_tag_str.strip(), _tag_str.strip())"></t>
+        <t t-if="not result_msg">
+            <t t-set="result_msg">Sorry, we could not find any<b>%s</b> results<b>%s</b>%s%s%s.</t>
```

Comment:
Mmmh... will lead to ugly translations, and a strange formatting below. Can't you do something more html-oriented, like removing the variable and putting the right content where it belongs ? Did not check the whole code but that's quite "low level" .